demos/toolbar/index.php

```diff
@@ -173,10 +173,10 @@
 
 		<div data-demo-html="true">
 			<div data-role="header">
-				<div data-role="controlgroup" data-type="horizontal" class="ui-mini ui-button-left">
-					<a href="#" class="ui-button">Add<span class="ui-icon ui-icon-plus"></span></a>
-					<a href="#" class="ui-button">Up<span class="ui-icon ui-icon-arrow-u"></span></a>
-					<a href="#" class="ui-button">Down<span class="ui-icon ui-icon-arrow-d"></span></a>
+				<div data-role="controlgroup" data-type="horizontal" class="ui-mini ui-toolbar-header-button-left">
+					<a href="#" class="ui-button ui-icon-end">Add<span class="ui-icon ui-icon-plus"></span></a>
```

Comment:
there is no longer a class `ui-icon-end` just place the icon where is should go and add a space to give it proper position